test/integration/dynamic-routing/test/index.test.js

```diff
@@ -1496,8 +1518,9 @@ function runTests({ dev }) {
             },
           },
           {
-            namedRegex:
-              '^/(?<nxtPname>[^/]+?)/(?<nxtPcomment>[^/]+?)/(?<nxtPrest>.+?)(?:/)?$',
+            namedRegex: normalizeRegEx(
```

Comment:
are these normalizations needed since we have a `.map(normalizeRouteRegExes(item)` below?

Reply:
Good catch updated